maven-scm-providers/maven-scm-providers-svn/maven-scm-provider-svnexe/src/test/java/org/apache/maven/scm/provider/svn/svnexe/command/remoteinfo/SvnExeRemoteInfoCommandTrailingSlashTckTest.java

```diff
@@ -0,0 +1,39 @@
+package org.apache.maven.scm.provider.svn.svnexe.command.remoteinfo;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.maven.scm.provider.ScmProviderRepository;
+import org.apache.maven.scm.provider.svn.repository.SvnScmProviderRepository;
+
+/**
+ * @author Bertrand Paquet
+ */
+public class SvnExeRemoteInfoCommandTrailingSlashTckTest
+    extends AbstractSvnExeRemoteInfoCommandTckTest
+{
+
+    @Override
+    protected ScmProviderRepository getScmProviderRepository()
+        throws Exception
+    {
+        return new SvnScmProviderRepository( "http://svn.apache.org/repos/asf/maven/maven-3/trunk/" );
```

Comment:
do we really want to depends on such external resources?

Reply:
That was in the original test. Actually we don't, but that should be addressed in a separate PR. There might be other spots as well.

Reply:
haha LOL I can even put a very expensive bet on who did that 🤣

Reply:
That's the cheap solution

Reply:
@olamy at work:
```
-------------------------------------------------------------------------------
Test set: org.apache.maven.scm.provider.svn.svnexe.command.remoteinfo.SvnRemoteInfoCommandTest
-------------------------------------------------------------------------------
Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 1,218.536 s <<< FAILURE! - in org.apache.maven.scm.provider.svn.svnexe.command.remoteinfo.SvnRemoteInfoCommandTest
testNotExist(org.apache.maven.scm.provider.svn.svnexe.command.remoteinfo.SvnRemoteInfoCommandTest)  Time elapsed: 609.31 s  <<< ERROR!
org.apache.maven.scm.ScmException: 
cmd.exe /X /C "svn --non-interactive ls http://svn.apache.org/repos/asf/maven/scm/trunk/foo/bar/beer@".The svn command failed:svn: E170013: Unable to connect to a repository at URL 'http://svn.apache.org/repos/asf/maven/scm/trunk/foo/bar/beer'
svn: E175012: Connection timed out

	at org.apache.maven.scm.provider.svn.svnexe.command.remoteinfo.SvnRemoteInfoCommandTest.testNotExist(SvnRemoteInfoCommandTest.java:53)

testExist(org.apache.maven.scm.provider.svn.svnexe.command.remoteinfo.SvnRemoteInfoCommandTest)  Time elapsed: 609.226 s  <<< ERROR!
org.apache.maven.scm.ScmException: 
cmd.exe /X /C "svn --non-interactive ls http://svn.apache.org/repos/asf/maven/scm/trunk/@".The svn command failed:svn: E170013: Unable to connect to a repository at URL 'http://svn.apache.org/repos/asf/maven/scm/trunk'
svn: E175012: Connection timed out

	at org.apache.maven.scm.provider.svn.svnexe.command.remoteinfo.SvnRemoteInfoCommandTest.testExist(SvnRemoteInfoCommandTest.java:42)
```

Reply:
yup maybe something with testcontainers using such image https://hub.docker.com/r/elleflorio/svn-server/ 
but frankly no much motivation to maintain more longer svn :( 
why not if really nothing else to do :)

Reply:
No, it jus thas to be `file:///`. It is more than sufficient for a test.

Reply:
From a Subversion perspective everything with a URL is remote because it is not a working copy.

Reply:
http://localhost/dsssdf is local also. Makes not difference.